### PR TITLE
Merge Bshortall fork into main

### PR DESCRIPTION
**Pull request by Bshortall**

### CODE REVIEW

**Comment by jonathanm-12** [comment]
Do we need the .vscode directory? It looks like it was set up just to run Python locally in vscode.

Also, please include the source code referenced for the Python programs.

File: Model/trained_models/epoch_FBC_000.pth
Comment (mitch-gilmore) [inline Model/trained_models/epoch_FBC_000.pth:1]: please remove from commit.

File: Model/trained_models/epoch_FBC_090.pth
Comment (mitch-gilmore) [inline Model/trained_models/epoch_FBC_090.pth:1]: Please remove from commit.

File: Model/trained_models/results.csv
Comment (mitch-gilmore) [inline Model/trained_models/results.csv:1]: please remove from commit

File: Model/.vscode/launch.json
Comment (mitch-gilmore) [inline Model/.vscode/launch.json:1]: please remove from commit

**Review by mitch-gilmore — CHANGES_REQUESTED** [review]

**Review by mitch-gilmore — APPROVED** [review]
looks good

**Review by jonathanm-12 — APPROVED** [review]

**Pull request merged**